src/components/NewComment.js

```diff
@@ -1,15 +1,42 @@
 import { useParams } from 'react-router-dom';
+import { postComment } from '../utils/api';
+import { useState } from 'react';
 
-function NewComment() {
-
+function NewComment({ comments, setComments }) {
+  const user = 'tickle122';
+  const [comment, setComment] = useState('');
+  const [error, setErrors] = useState(false);
+  const [errorMsg, setErrorMsg] = useState('');
   const { review_Id } = useParams();
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (comment[0] === undefined) {
+    } else {
+      postComment(review_Id, user, comment).then((response) => {
+        setComments([response.comment, ...comments]);
+        setErrors(false).catch((err) => {
```

Comment:
This catch block should really be aiming to catch if the API call postComment has worked or not, so should be chained onto the postComment function, I think it's currently causing an uncaught promise to happen when you try to submit an empty comment in the console.